root_VS2012/programs/C#/Frameworks/Infrastructure/Public/Db/DamSqlSvr.cs

```diff
@@ -114,6 +114,11 @@ public SqlConnection DamSqlConnection
                 // コネクションを戻す
                 return _cnn;
             }
+            set
+            {
+                //set the connection
+                _cnn = value;
+            }
```

Comment:
@yusukemaegawa 

Set the **SqlConnection** for the **DataDependentRouting** from here.